net/tcp/tcp_input.c

```diff
@@ -737,12 +736,14 @@ static void tcp_input(FAR struct net_driver_s *dev, uint8_t domain,
             }
         }
 
+#ifdef CONFIG_NET_TCP_WRITE_BUFFERS
       /* Update sequence number to the unacknowledge sequence number.  If
        * there is still outstanding, unacknowledged data, then this will
        * be beyond ackseq.
        */
 
-      sndseq = tcp_getsequence(conn->sndseq);
+      uint32_t sndseq = tcp_getsequence(conn->sndseq);
```

Comment:
need keep the line at line 687 since C89 doesn't support to define variables in the middle of code block.

Reply:
Thank you. I returned the line to the place.